new-dti-website/src/app/page.tsx

```diff
@@ -1,10 +1,13 @@
 import { Button } from '../../components/ui/button';
+
 import Navbar from '../../components/navbar';
+import Footer from '../../components/footer';
 
 const Home = (): JSX.Element => (
   <div>
```

Comment:
`Navbar` and `Footer` actually should not be in the `Home` component, since they apply to all pages. We should actually remove both components here.